conanfile.py

```
@@ -0,0 +1,98 @@
+
+    def _benchmark_enabled(self):
+        return self.options.target in ("benchmark", "all")
+
+    def _profiling_enabled(self):
+        if not self._benchmark_enabled():
+            return False
+        return self.options.profiling
+
+    def validate(self):
+        if not any(remote.name.lower() == "4j-company" for remote in ConanAPI().remotes.list()):
+            raise Exception("Required remote '4J-company' is missing. Please add it manually. For details, see https://github.com/4J-company/conan-center-index")
```

Ruff (0.11.9)

Create your own exception

(TRY002)